docs/resources/region.md

```diff
@@ -1,35 +1,55 @@
----
-# generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "biganimal_region Resource - terraform-provider-biganimal"
-subcategory: ""
-description: |-
-  Manage a region
----
-
 # biganimal_region (Resource)
-
-Manage a region
-
-
+The region resource is used to manage regions for a given cloud provider. See https://www.enterprisedb.com/docs/biganimal/latest/getting_started/activating_regions/ for more details.
```

Comment:
```suggestion
The region resource is used to manage regions for a given cloud provider. See [Activating regions }(https://www.enterprisedb.com/docs/biganimal/latest/getting_started/activating_regions/) for more details.
```

Reply:
Clickable link is added at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/0aea7392df32d1d3b7fcb6e35524aa585fa8a660